dev/tools/gen_defaults/lib/fab_template.dart

```diff
@@ -13,7 +13,30 @@ class FABTemplate extends TokenTemplate {
 class _M3Defaults extends FloatingActionButtonThemeData {
   _M3Defaults(this.context, this.type, this.hasChild)
     : _colors = Theme.of(context).colorScheme,
-      _textTheme = Theme.of(context).textTheme;
+      _textTheme = Theme.of(context).textTheme,
+      super(
+        elevation: ${elevation("md.comp.fab.primary.container.elevation")},
```

Comment:
not a comment for this PR but a general observation: things would be faster if we could use constants with integer values rather than strings for keys and suchlike.

Reply:
Point taken. However, this is in a template that we use to generate the defaults, so is only run when we pull in new values from the token database (i.e. rarely and manually). The keys are directly matching the keys from a JSON files we extract from the token database. It just didn't seem worth the time to remap these key strings to some other constants just for the generation, as it is pretty fast to generate the templates as is. If this was in more performant critical code I would certainly take the time to optimize it.